src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -1019,23 +977,170 @@ const { items: itemsPage2 } = res.data.listTodos;
 </Block>
 </BlockSwitcher>
 
-A `nextToken` is a very long string that looks like `"eyJ2ZXJzaW9uejE1a2RPanZPQzFCMlFTdGNxdUFBQUJxekNDQWFjR0NTcUdTSWIzRFFFSEJxQ0NBWmd3Z2dHVUFnRUFNSUlCalFZSktvWklodmNOQVFjQk1CNEdDV0NHU0FGbEF3UUJMakFSQkF5eEtWQjUvTlJxS2o5ZHBYc0NBUkNBZ2dGZUZFNW1MbTRkb25BOUg0U0FpOGhSZ1lucmRndmQz"`, which represents the cursor to the starting item of the next query made with these filters.
-[question]: # (RENE, can we omit the example of the long string and rewrite this as: A `nextToken` is a very long string that represents the cursor to the starting item of the next query made with these filters.)
+<Accordion title='Implement pagination in your GraphQL API' headingLevel='3' eyebrow='Walkthrough'>
```

Comment:
Moved the guide info here to an expander since we noted it is still relevant and one of the goals was to eventually move all guide content to libraries. However, we may want to review where this may also live on other JTBD Guides.